stacks/container-apps/cms.prx.org.yml

```diff
@@ -0,0 +1,459 @@
+# stacks/container-apps/cms.prx.org.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Docker web application with optional worker, using the shared platform ALB
```

Comment:
Do you planning to change this, and other copied-from-shared-stack things that don't need to be so specific?

Or are you just going for minimal changes after copy? I see you did leave all the configs in the parent stack, rather than moving them into here.

(And going forward, will we tend to keep configs in the lower/root stacks, or push them up to the specific app?)

Reply:
Leaving as is for me, will clean up the unnecessary parameters down the road.